### DISCUSSION TITLE
Tooltip click behavior hiding tooltip

### DISCUSSION THREAD

**Discussion by Skim-chee** [radix-ui/primitives#616] [Help] [answered]
We noticed that clicking the Tooltip trigger was hiding the tooltip
Ex from the radix doc site: https://i.gyazo.com/dc7681426e4077d7e7ee17ba32bd5bec.mp4 

Our team was wondering if this was intentional? 

Seemed a bit unintuitive with the default settings (esp. with a delay in place) - for ex, user hovers over the tooltip => sees nothing popup for a short bit => clicks it 

^ this clicked active state will prevent the tooltip then from ever showing up until the user exits focus from the Tooltip trigger 

Thanks in advance!

**Comment by benoitgrelard** [comment, marked as answer]
Hi @Skim-chee,

This is generally accepted behaviour for a tooltip, ie. it will disappear once the tool has been interacted with.
It is also the behaviour of the native `title` attribute tooltip.

**Reply by Skim-chee** [reply]
Thank you @benoitgrelard! Makes more sense when compared with the `title` attribute
